src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddMimeTypeGuesserPass.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Registers custom mime types guessers.
```

Comment:
We'd need to change all occurrences of "mime" word to "MIME" (in PHPdoc, not in actual code) because that's how it's spelled (https://en.wikipedia.org/wiki/MIME).

Reply:
done